airgun/entities/computeresource.py

```diff
@@ -59,6 +61,25 @@ def vm_poweroff(self, rhev_name, vm_name):
         if vm['Power'].widget.read() == 'On':
             vm['Actions'].widget.click(handle_alert=True)
 
+    def list_computeprofiles(self, entity_name, computeprofile_name):
+        view = self.navigate_to(self, 'Detail', rhev_name=entity_name)
+        view.compute_profiles.table.row(compute_profile=computeprofile_name)[
+            'Compute profile'].widget.click()
+        view.submit.click()
+
+    def update_computeprofile(self, entity_name, computeprofile_name, values):
+        view = self.navigate_to(self, 'Detail', rhev_name=entity_name)
+        view.compute_profiles.table.row(compute_profile=computeprofile_name)[
+            'Compute profile'].widget.click()
+        view.fill(values)
```

Comment:
your `view` is equal to `self.navigate_to(self, 'Detail', rhev_name=entity_name)` which is compute resource view, so how do you actually perform `update_computeprofile` here?
and read in the next?

Reply:
yes, you can click on table item to get on compute_profile page, but that's it

Reply:
You can actually update compute profile here because this method do this:
1. Select the created ec2 CR.
2. Click Compute Profile tab.
3. Click (3-Large) compute profile
4. Fill values 
           ('flavor': AWS_EC2_FLAVOR_T2_MICRO,
            'availability_zone': availability_zone,
            'subnet': subnet,
            'managed_ip': managed_ip,)
5. Submit

And it is same with read but it doesn't fill in values, just read value of subnet (as it is my test in robottelo)